Sources/Web3Modal/Screens/ConnectWallet/ConnectWalletView.swift

```diff
@@ -10,7 +10,7 @@ struct ConnectWalletView: View {
     var wallets: [Wallet] {
         
         var uniqueValues: [Wallet] = []
-        (store.recentWallets + store.featuredWallets).forEach { item in
+        (store.recentWallets + store.featuredWallets + store.customWallets).forEach { item in
```

Comment:
It calculate every time your reading from `wallets` property. Is it fine?

Reply:
We are talking  5-10 wallets total here, and it's really simple read and write. So I'm not expecting issues here.